feathr-example/scripts/upload_home_credit_data.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash          
+
+# Fill in details in this section
+subscription_id="<SUBCRIPTION_ID>"
+resource_prefix="<RESOURCE_PREFIX>"
+location="eastus"
+synapse_sql_admin_name="cliuser1"
+synapse_sql_admin_password="<PASSWORD>"
+synapse_sparkpool_name="spark31"
```

Comment:
Nit: maybe we can remove these default names? or maybe give a more generic name and remove the trailing numbers

Reply:
Changed this one. Thanks.